adminapi/src/main/java/io/minio/admin/model/TierStats.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage,
+ * (C) 2022 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio.admin.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Per-tier stats of a remote tier.
+ *
+ * @see https://github.com/minio/minio/blob/master/cmd/data-usage-cache.go#L102
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TierStats {
+  @JsonProperty("TotalSize")
+  private long totalSize;
```

Comment:
You could make this fields as `public` to avoid getters.

Reply:
I want to avoid this approach as much as I can. Not only it's against all best practices in Java, but it would make the whole minio-java API inconsistent. For example https://github.com/minio/minio-java/blob/master/api/src/main/java/io/minio/messages/Stats.java doesn't expose any literals directly, only via getters. Same with the other object types - only retrievable via getters and there is no option to actually change them.

I've changed the `getTotalSize()` and all other getters in all messages into the same structure as it's used in the `api` module -> `totalSize()` (so the `get` part is omitted), also I've made all collections returned from those getters unmodifiable, to be consistent with the approach already set here. 

Let me know if it's ok this way or not. Thanks